lib/email/styles.rb

```diff
@@ -228,7 +228,7 @@ def replace_relative_urls
 
       @fragment.css('[href]').each do |element|
         href = element['href']
-        if href =~ /^\/\/#{host}/
+        if href.start_with?("\/\/#{host}")
```

Comment:
I'm curious, what's the perf improvement of using `start_with?` as opposed to the regex?